Header header
uint32 leftRed
uint32 leftGreen
uint32 leftBlue
uint32 rightRed
uint32 rightGreen
uint32 rightBlue

